### PR TITLE
Add R as experimental and PHP as GA

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

**Comment by pabloest** [comment]
Thanks for this PR, @adamkvitek.

Can you also edit the docs home page (index.md) with the same info? (see line 16 of this supported-languages.md file)

**Comment by adamkvitek** [comment]
@pabloest updated... and gave a little bit more attention to the comments.

**Review by raghavjain3 — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ Semgrep is developed and commercially supported by [r2c, a software security com
 
 Semgrep supports 20+ languages.
 
-<!--  coupling: if you modify this table, copy paste it also to status.md -->
-<!--  for readability, please keep each column in alphabetical order -->
+<!--  coupling: If you modify this table, copy paste it also to supported-languages.md! -->
```
[the remark this answers is not in the document]
Comment (pabloest) [reply docs/index.md:48]: @adamkvitek Yes, thanks for updating the comment also!

[the remark this answers is not in the document]
Comment (adamkvitek) [reply docs/index.md:48]: Ok, so this was truly the correct file! :-)

**Pull request merged**